action.yml

```diff
@@ -0,0 +1,63 @@
+name: "get-aws-account-id"
+description: "get-aws-account-id-gh-action"
+inputs:
+  account-prefix:
+    description: "Identification of baseline - usually ohp (ohpen cloud) or dvc (davinci cloud)"
+    required: true
+  client:
+    description: "Name of the client"
+    required: true
+  stage:
+    description: "Stage (dev, tst, acc, prd)"
+    required: true
+
+  aws-access-key:
+    description: "AWS access key for accessing organizations"
+    required: true
+  aws-secret-key:
+    description: "AWS secret key for accessing organizations"
+    required: true
+  aws-role-arn:
+    description: "AWS role arn for accessing organizations"
+    required: true
+
+outputs:
+  aws-account-id:
+    description: "AWS account id"
+    value: ${{ steps.retrieve.outputs.id }}
+  aws-account-tag:
+    description: "AWS account tag used for retrieving id"
+    value: ${{ steps.retrieve.outputs.tag }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Set up aws credentials
+      shell: bash
+      run: |
+        aws configure set aws_access_key_id ${{ inputs.aws-access-key }}
```

Comment:
maybe different profile than default, composite action I think should not reset your default profile

Reply:
done